scripts/qa_crowbarsetup.sh

```diff
@@ -3331,7 +3327,7 @@ function oncontroller_upload_defcore
     # run only the specified tests
     tempest run --whitelist-file defcore.txt
     source /root/.openrc
-    ostestr last --subunit --no-pretty | subunit-2to1 > tempest.subunit.log
```

Comment:
If we don't have `subunit-2to1` command, this code should be fixed, right?

Reply:
We don't have subunit-1to2.
The reverse is still there

Reply:
Oh, seriously. I don't know what's the criteria of the selection..

Anyway, I think the `last` subcommand is wrong, and `ostestr` runs the tempest tests again. And I think it doesn't run the `defcore.txt`.

Do I miss something?

Reply:
Sorry for the delay. I just clarified and rechecked some of my statements.

You are right ostestr line can be removed completely. We used it only to generate the subunit report of the last tempest run. But it does not do that, so can be removed.

However we still need a subunit version 1 report for the refstack client.
And `testr last --subunit` generates subunit version 2 report, but we need version 1. 

Since neither subunit-2to1 nor subunit-1to2 filter is available after single-spec switch, i propose the following added after you remove the ostestr line:
```bash
zypper ar http://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP3/devel:languages:python.repo
    zypper in python3-testtools python3-subunit
    testr last --subunit | subunit-2to1 > tempest.subunit_1.log
```

Reply:
Hi! Thank you for your reply :)
I think this PR fixes the issue. Do you think this is still not enough or wrong direction to fix?

Reply:
Yes, the PR fixes the issue.
But please remove the  `subunit-2to1` , because it does not exist.

Leave only `testr last --subunit > tempest.subunit.log`

Thank you.